### ISSUE TITLE
[ui5-slider]: redundant role and attributes presented on slider's scale

### ISSUE THREAD

**Issue by niyap** [UI5/webcomponents#13892]
### Bug Description

As the progress bar of ui5-slider is not focusable, according to the accessibility specification it should not have role="slider" and all the other attributes related to that role. Currently, they are presented and have to be removed.

### Affected Component

ui5-slider

### Expected Behaviour

role attribute as well as all the other accessibility attributes related to that role should not be presented on the slider's scale, only on the handle.

### Steps to Reproduce

1.
2.
3.
...

### Priority

Medium

### UI5 Web Components Version

latest

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
